static/manifest.webmanifest

```
@@ -0,0 +1,22 @@
+{
+  "short_name": "Kubernetes",
+  "name": "Kubernetes Documentation",
```

To me, “Kubernetes” works fine as a long and a short name.
```suggestion
  "name": "Kubernetes",
```